src/features/store.ts

```diff
@@ -177,20 +188,24 @@ async function initWarpContext({
   registry: IRegistry;
   chainMetadataOverrides: ChainMap<Partial<ChainMetadata> | undefined>;
   warpCoreConfigOverrides: WarpCoreConfig[];
+  warpDeployConfig: WarpDeployConfigChainAddressMap;
 }) {
   try {
-    const coreConfig = await assembleWarpCoreConfig(warpCoreConfigOverrides);
-    const chainsInTokens = Array.from(new Set(coreConfig.tokens.map((t) => t.chainName)));
     // Pre-load registry content to avoid repeated requests
     await registry.listRegistryContent();
+    const { warpCoreConfig: coreConfig, warpDeployConfig } = await assembleWarpCoreConfig(
```

Comment:
Do we need to move the assembly of the warpDeployConfig inside `assembleWarpCoreConfig`? Imo the name is not reflective